README.md

```diff
@@ -1,26 +1,29 @@
 # WooriDB
 
-WooriDB is a general purpose (**EXPERIMENTAL**) time serial database, which means it contains all entities registries indexed by DateTime. It is schemaless, key-value storage and uses its own query syntax that is similar to SparQL and Crux's Datalog. Some other features are:
+WooriDB is a general purpose (**EXPERIMENTAL**) time serial database, which means it contains all entities registries indexed by DateTime. It is schemaless, key-value storage and uses its own query syntax that is similar to SparQL and Crux's Datalog. 
+
+Some other features are:
 - Hashing keys content with [`ENCRYPT`](https://github.com/naomijub/wooridb#create-entity) keyword.
 - Hashed values are filtered out and can only be checked with  [`CHECK`](https://github.com/naomijub/wooridb#checks-validity-of-of-an-encrypted-key) keyword.
 - [`Ron`](https://github.com/ron-rs/ron/blob/master/docs/grammar.md) schemas for input and output.
   - [ ] JSON to be supported via feature.
   - [ ] EDN to be supported via feature.
 - Entities are indexed by `entity_name` and `Uuid`. Entity format is a HashMap where keys are strings and values are supported [`Types`](https://github.com/naomijub/wooridb/blob/main/wql/src/lib.rs#L78).
 - Stores persistent data locally.
-  - [ ] `S3` as backend to be devloped.
-  - [ ] `Postgres` as backend to be devloped.
-  - [ ] `DynamoDB` as backend to be devloped.
-- Is able to handle very large numbers when ending with letter `P`, `98347883122138743294728345738925783257325789353593473247832493483478935673.9347324783249348347893567393473247832493483478935673P`.
+  - [ ] `S3` as a backend is to be developed.
+  - [ ] `Postgres` as a backend is to be developed.
+  - [ ] `DynamoDB` as a backend is to be developed.
+- Able to handle very large numbers when using the `P` suffix.
+  - Ex: `98347883122138743294728345738925783257325789353593473247832493483478935673.9347324783249348347893567393473247832493483478935673P`.
 - Configuration is done via environment variables.
-  - [ ] Non sensible configurations with `Config.toml`.
+  - [ ] Non sensitive configurations are done with `Config.toml`.
   - [ ] CORS
 - Authentication and Authorization via session token
   - [ ] Creating and removing ADMINs/new users.
 - [Conditional Update](https://github.com/naomijub/wooridb#match-update-entity)
 - [ ] Possible Relation Algebra
 
-`Woori` means `our` and although I developed this DB initially alone, it is in my culture to call everything that is done for our community and by our community **ours**.
+`Woori` (우리) means `our` and although I developed this DB initially alone, it is in my culture to call everything that is done for our community and by our community **ours**.
```

Comment:
I just don't want it to have the hangul version. This is a personal reason, as well as it has a similar sounding to altaic tatar language that has sounds that variates from Biz to Bwoozi (also personal, but way less known).